youtube_dl/extractor/mixcloud.py

```diff
@@ -122,3 +120,31 @@ def _real_extract(self, url):
             'view_count': view_count,
             'like_count': like_count,
         }
+
+class ServerAttemptSequence:
```

Comment:
@jaimeMF I agree, my way is not very elegant but I couldn't make it any simpler. My algorithmic skills are a bit rusty also :laughing: (where's the WD40 when you need it?). Your alternatives do look nicer but they don't work as expected.

With `start = 2` and `max_value = 5`, both generate:
```
2, 3, 1
```

Where I expect:

```
2, 3, 1, 4, 5
```

Do you have other suggestions?